plugin/src/main/java/org/opensearch/ml/utils/RestActionUtils.java

```diff
@@ -241,7 +261,8 @@ private static boolean isAdminDN(String dn) {
         try {
             return isAdminDN(new LdapName(dn));
         } catch (InvalidNameException e) {
-            return false;
+            boolean isAdmin = adminUsernames.contains(dn);
+            return isAdmin;
```

Comment:
Nit: can directly `adminUsernames.contains(dn);` rather than first assigning to a variable?